dockerfiles/theia/docker/ubi8/builder-post-clone.dockerfile

```diff
@@ -1,3 +1,7 @@
 # patch electron module by removing native keymap module (no need to have some X11 libraries)
 RUN line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) && \
-    sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json
+    if [[ ${line_to_delete} ]]; then \
```

Comment:
In which case it's not found ?

Reply:
several times I've run crw-theia and the line must have already been patched in upstream so the re-removal isn't needed. 

```
2019-12-09 21:03:07,471 - atomic_reactor.plugins.imagebuilder - INFO - --> RUN line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) &&     sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json
2019-12-09 21:03:08,536 - atomic_reactor.plugins.imagebuilder - INFO - sed: -e expression #1, char 1: unknown command: `,'
2019-12-09 21:03:08,837 - atomic_reactor.plugins.imagebuilder - INFO - running 'line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) &&     sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json' failed with exit code 1
2019-12-09 21:03:08,838 - atomic_reactor.plugin - ERROR - Build step plugin imagebuilder failed: image build failed (rc=1): running 'line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) &&     sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json' failed with exit code 1

2019-12-09 21:03:08,838 - atomic_reactor.inner - ERROR - buildstep plugin failed: image build failed (rc=1): running 'line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) &&     sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json' failed with exit code 1

2019-12-09 21:03:08,838 - atomic_reactor.inner - DEBUG - caught exception (PluginFailedException: image build failed (rc=1): running 'line_to_delete=$(grep -n native-keymap ${HOME}/theia-source-code/dev-packages/electron/package.json | cut -d ":" -f 1) &&     sed -i -e "${line_to_delete},1d" ${HOME}/theia-source-code/dev-packages/electron/package.json' failed with exit code 1
) so running exit plugins
```

--- https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/get-sources-rhpkg-container-build/2324/consoleText